extensions/typescript-language-features/src/utils/configuration.ts

```diff
@@ -103,6 +103,16 @@ export class TypeScriptServiceConfiguration {
 	public readonly watchOptions: protocol.WatchOptions | undefined;
 	public readonly includePackageJsonAutoImports: 'auto' | 'on' | 'off' | undefined;
 	public readonly enableTsServerTracing: boolean;
+	public readonly includeInlineParameterNameHints: boolean | undefined;
```

Comment:
You shouldn't need to add these new preferences here. These preferences are the global ones for  running the TS server